tests/assertions.yml

```diff
@@ -33,6 +33,17 @@
       - check_ruby_multiple.rc == 0
     msg: No multiple rubies installed
 
+- name: Check that ruby-{{ rvm_tests_mri_version_to_be_deleted }} is not present
```

Comment:
I find it safer to have a first assertion before that, to verify that the ruby to be deleted is actually present. Without it, the risk that the test will pass, but actually without protecting us from regression, is present.

Reply:
@thbar I'm not sure I understand your point about protecting from regression here, if you wouldn't mind providing an example.

Reply:
@thbar I am not sure to understand you either. I suppose you would like to first assert that `rvm_tests_mri_version_to_be_deleted` was once installed on the system, to prove that its current absence was caused by the ansible role. This is but tricky to implement with the current tests setup, as the assertions are executed on the "final" state.

Reply:
@gildegoma @danochoa putting my previous comment (sent via email reply, not sure you got it) here: It is exactly what @gildegoma. This will avoid a risk of false positive.

But let’s forget about it for now, it will be better handled with a more advanced testing system, if we move to that. This cannot be tested correctly if we assert on a single global final state.